### PR TITLE
Update

### PR DESCRIPTION
**Pull request by damdafayton**
In this project I used a relational database to create the initial data structure for a vet clinic. I created a table to store animals' information, insert some data into it, and query it.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /* Database schema to keep the structure of entire database. */
+-- HELLO HELLO HELLO
```
Comment (Mihndim2020) [inline schema.sql:3]: - [ ] Please kindly update the data type of `weight_kg`, for this particular project, the appropriate data type is `decimal`. Something like so; `weight_kg decimal`

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @damdafayton, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (damdafayton) [reply schema.sql:3]: Hi @Mihndim2020,
`numeric` and `decimal` are the same according to [PostgreSQL documentation.](https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-NUMERIC-DECIMAL)

numeric

**Review by Gambit142 — APPROVED** [review]
Hi @damdafayton 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Optional Suggestion

- In your queries.sql, kindly change the query for the case _**it can list the name of all animals born between 2016 and 2019**_ from 
`select * from animals where date_of_birth between '2016-01-01' and '2019-01-01'` to 
`select * from animals where date_of_birth between '2016-01-01' and '2019-12-31'`. Remember you are querying from the beginning of the year 2016 to the end of the year 2019. 👍 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**